src/tty/windows.rs

```diff
@@ -225,13 +233,78 @@ impl RawReader for ConsoleRawReader {
                 let c = rc?;
                 KeyEvent::new(c, mods)
             };
+
             debug!(target: "rustyline", "wVirtualKeyCode: {:#x}, utf16: {:#x}, dwControlKeyState: {:#x} => key: {:?}", key_event.wVirtualKeyCode, utf16, key_event.dwControlKeyState, key);
+
+            if self.enable_bracketed_paste {
+                if esc_seq_len == 0 && key == KeyEvent(K::Esc, M::NONE) {
+                    esc_seq[esc_seq_len] = '\x1b';
+                    esc_seq_len += 1;
+                    continue;
```

Comment:
What about a single ESC ?

Reply:
Yes, you're right.  It seems that, after setting virtual mode, all arrow keys etc. turn into VT escape codes.

Therefore I'll have to add all those VT escape code processing back in.

I suppose, in that case, a single Esc won't be possible unless we do a timeout...

Reply:
OK, new additions that implement most of the relevant escape sequences...  Except for a lone `Esc` of course...

Reply:
I am sorry but these changes are too invasive (just to support bracketed paste on windows terminal).

Reply:
I have refactored the changes into a separate struct and now a single ESC can also be detected.  Please let me know if there is anything else you think is blocking?

Reply:
Sorry but for me we don't go into the right direction.
- if we need to activate `ENABLE_VIRTUAL_TERMINAL_INPUT` to do bracketed paste, we must factorise code between unix and windows like `termwiz` and its [InputParser](https://github.com/wez/wezterm/blob/main/termwiz/src/terminal/windows.rs#L842-L843) and make the [distinction](https://github.com/wez/wezterm/blob/main/termwiz/src/terminal/windows.rs#L623-L628) between "standard" console and windows terminal.
- there is already a [poll function](https://github.com/kkawakam/rustyline/pull/539/files#diff-e734d9d9fe3d302f6650d28680f8903112ac3f3d0e102081ae26ae4ec28363e3R241) waiting to be merged.

Reply:
Gotcha.  That makes sense.  It makes sense to reuse the escape codes processing from Unix and to consolidate into one single code base for VT-like terminals.

Reply:
I feel sorry because you have an issue, you solved it but I would prefer a better solution...
Thanks for your comprehension.